star-wars-app/UI/Label.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Label.swift
+//  star-wars-app
+//
+//  Created by Michał Nowak on 13/01/2020.
+//  Copyright © 2020 Michal Nowak. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class Label: View {
+    // Base
+    static func baseLabelStyle(_ label: UILabel) {
```

Comment:
I'm wondering why doesn't this return style as well. You could then compose it with other styles, and now you need to pass a label instead.